configuration.packages.json

```diff
@@ -2,7 +2,7 @@
   "packages": {
     "EdFiMinimalTemplate": {
       "PackageName": "EdFi.Suite3.Ods.Minimal.Template",
-      "PackageVersion": "6.1.1",
+      "PackageVersion": "6.1.40",
```

Comment:
There look to be conflicts with this file that would prevent it from being merged to main currently. I also noticed how the initdev builds are failing which I think are related to this not being up to date as well. So I think what might need to happen is updating this from main, and then doing the usual build chain of updating these templates accorindgly.